lms/envs/common.py

```diff
@@ -1247,7 +1247,8 @@
     'js/vendor/requirejs/require.js',
     'js/RequireJS-namespace-undefine.js',
     'js/vendor/URI.min.js',
-    'js/vendor/backbone-min.js'
+    'js/vendor/backbone-min.js',
+    'js/vendor/afontgarde/modernizr.fontface-generatedcontent.js',
```

Comment:
This modernizr file is used by the edxicons and the new accessible icon fonts. While they're not yet implemented throughout the platform, this paves the way and allows us to start converting them as we need. I added it to the global list because it needs to be available on every page in the platform.

Do you have another suggestion for global scripts like this?